source/agora/node/Ledger.d

```diff
@@ -379,7 +379,7 @@ public class Ledger
         const expect_height = Height(this.getBlockHeight() + 1);
         auto utxo_finder = this.utxo_set.getUTXOFinder();
 
-        foreach (tx; data.tx_set)
+        foreach (const ref tx; data.tx_set)
```

Comment:
It doesn't seem relevant to the first commit.
Move to the second commit right ?